README.md

```diff
@@ -4,14 +4,67 @@ Library for managing [WebMock][] fixtures.
 
 [WebMock]: https://github.com/bblimke/webmock
 
-## Installation
+[Documentation](http://www.rubydoc.info/github/underdogio/webmock-fixtures).
 
+## Installation
 `gem install webmock-fixtures`
 
 or with Bundler
 
 `gem 'webmock-fixtures`
 
+## Getting Started
+WebMock fixtures provides a way to pre-define all of your fixtures ahead of time and start them easily during tests.
+
+```ruby
+require "httparty"
+require "webmock/fixtures"
+
+# Register a fixture as `:get_example`
+WebMock::Fixtures::Manager.register_fixture(
+  :get_example, :get, %r{www.example.org}, :body => "Hello World")
+
+# Create a manager with the `:get_example` fixture started
+manager = WebMock::Fixtures::Manager.run([:get_example])
+
+# Make our HTTP request
+response = HTTParty.get("http://www.example.org")
+
+# Assert we received our mock response
+if response.body != "Hello World"
+  fail("Response body \"#{response.body}\" was not what we expected")
+end
+
+# Access the created `WebMock::RequestStub` for our `:get_example` fixture
+stub = manager[:get_example]
```

Comment:
Unfortunately a method like that doesn't exist.

`WebMock` really should be used with a test framework like `RSpec`, since it provides matchers for things like `expect(stub).to(have_been_called)`.